packages/jest-matcher-utils/src/index.ts

```diff
@@ -517,29 +548,30 @@ export const matcherHint = (
 ): string => {
   const {
     comment = '',
-    expectedColor = EXPECTED_COLOR,
+    expectedColor = options.expectedColor ?? EXPECTED_COLOR,
     isDirectExpectCall = false, // seems redundant with received === ''
     isNot = false,
     promise = '',
-    receivedColor = RECEIVED_COLOR,
+    receivedColor = options.receivedColor ?? RECEIVED_COLOR,
     secondArgument = '',
-    secondArgumentColor = EXPECTED_COLOR,
+    secondArgumentColor = options.secondArgumentColor ?? EXPECTED_COLOR,
+    commonColor = options.commonColor ?? DIM_COLOR,
```

Comment:
How do you see the usage of `commonColor` option? 

I am not sure if it is useful to configure the dimmed style. The name is misleading, this is not a color, but a style (like bold or italic). For reference, ANSI escape codes for [colors](https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit) and [other params](https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_(Select_Graphic_Rendition)_parameters) (number 3 is dim).

The only replacement which makes sense for my eye – Normal intensity. Or simply no dim. Perhaps it would be enough to have a boolean setting?